zipabox.js

```diff
@@ -141,16 +143,16 @@ class Zipabox{
     return new Promise(function(resolve, reject){
       var attributeRequest = '?network=false&device=true&endpoint=false&clusterEndpoint=false&definition=false&config=false&room=false&icons=false&value=false&parent=false&children=false&full=false&type=false';
       this.debug && this.log("Methode getDeviceUUID()");
-      //this.debug && this.log("URL device :",this.baseURL + 'attributes/' + attributeUUID + attributeRequest);
+      // this.debug && this.log("URL device :",this.baseURL + 'attributes/' + attributeUUID + attributeRequest);
       // Check if uuid is a device or not
         // TODO ADD CHECK METHOD
       // Get the id with fetch
-      fetch(this.baseURL + 'attributes/' + attributeUUID + attributeRequest,myInitGet)
+      fetch(this.baseURL + 'attributes/' + attributeUUID + attributeRequest,myInitGet)	   
       .then(fstatus)
       .then(fjson)
       .then(function giveDeviceUUID(jsonResponse){
-        console.log("Response of getDeviceUUID. UUID source :", attributeUUID);
-        console.log("Device UUID : ",jsonResponse.device.uuid);
+        this.debug && this.log("Response of getDeviceUUID. UUID source :", attributeUUID);
+        this.debug && this.log("Device UUID : ",jsonResponse.device.uuid);
```

Comment:
Pour moi le "this.debug" n'est pas connu dans le cadre de cette fonction. L'as-tu testé ?

Reply:
oui il me semble que cela fonctionnait mais sinon pas de soucis pour desactiver ca c'était pour le debug ...